### PR TITLE
🚀11.3.0

### PR DESCRIPTION
**Pull request by slntopp**

### CODE REVIEW

File: charts/mondoo-operator/templates/metrics-service.yaml-z
Comment (imilchev) [inline charts/mondoo-operator/templates/metrics-service.yaml-z:1]: I believe we never commit the -z files. @czunker can you confirm?

File: charts/mondoo-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -29,10 +29,12 @@ spec:
   replicas: {{ .Values.controllerManager.replicas }}
   selector:
     matchLabels:
+      app.kubernetes.io/name: mondoo-operator
     {{- include "mondoo-operator.selectorLabels" . | nindent 6 }}
   template:
     metadata:
       labels:
+        app.kubernetes.io/name: mondoo-operator
```
Comment (imilchev) [inline charts/mondoo-operator/templates/deployment.yaml:37]: Can we keep this one? I thought we had to remove it

Comment (imilchev) [reply charts/mondoo-operator/templates/deployment.yaml:37]: yup, I can confirm, these should be gone. I don't have them when I generate the release. Perhaps you missed to change some `sed` to `gsed`?

Comment (czunker) [reply charts/mondoo-operator/templates/deployment.yaml:37]: Here is the command which should remove it: https://github.com/mondoohq/mondoo-operator/blob/main/Makefile#L372

**Review by czunker — APPROVED** [review]
LGTM

Thanks @slntopp

**Pull request merged**